flaml/automl.py

```diff
@@ -3174,18 +3183,34 @@ def _search(self):
                     from sklearn.ensemble import StackingClassifier as Stacker
                 else:
                     from sklearn.ensemble import StackingRegressor as Stacker
+                if self._use_ray is not False:
+                    import ray
+
+                    n_cpus = (
+                        ray.is_initialized()
+                        and ray.available_resources()["CPU"]
+                        or os.cpu_count()
+                    )
+                else:
+                    n_cpus = os.cpu_count()
+                n_jobs = (
+                    -self._state.n_jobs
+                    if abs(self._state.n_jobs) == 1
```

Comment:
why n_jobs = -self._state.n_jobs? This is unclear, can you add comments?

Reply:
so when n_jobs is max, minimize it; maximize it when min? why?

Reply:
Unresolved.

Reply:
What about now?